config/environments/production.rb

```diff
@@ -17,6 +17,11 @@
   config.i18n.fallbacks = true
   config.active_support.deprecation = :notify
   config.log_formatter = ::Logger::Formatter.new
+  config.web_socket_server_url = "wss://#{ENV['APPLICATION_HOST']}/cable"
+  #config.action_cable.allowed_request_origins = [
```

Comment:
Incorrect indentation detected (column 2 instead of 3).<br>Missing space after #.